### PR TITLE
b#48614111:  Add "break on error" setting to the top of powershell scripts in WASDK

### PR DESCRIPTION
**Pull request by alexlamtest**
b#48614111:  Add "break on error" setting to the top of powershell scripts in WASDK
With this PR, "break on error" is enabled everywhere except for 2 small blocks in 2 ps1 files, which are being tracked by 2 separate bugs.

//////////

A microsoft employee must use /azp run to validate using the pipelines below.

WARNING:
Comments made by azure-pipelines bot maybe inaccurate.
Please see pipeline link to verify that the build is being ran.

For status checks on the main branch, please use TransportPackage-Foundation-PR
(https://microsoft.visualstudio.com/ProjectReunion/_build?definitionId=81063&_a=summary)
and run the build against your PR branch with the default parameters.

### CODE REVIEW

File: tools/TerminalVelocity/Generate-TerminalVelocityFeatures.ps1 [hunk 1]
```diff
@@ -46,6 +46,8 @@ Param(
     [string]$Namespace
 )
 
+Set-StrictMode -Version 3.0
```
Comment (DrusTheAxe) [inline tools/TerminalVelocity/Generate-TerminalVelocityFeatures.ps1:49]: Why not also `$ErrorActionPreference = 'Stop'`

**Review by DrusTheAxe — APPROVED** [review]

File: tools/TerminalVelocity/Generate-TerminalVelocityFeatures.ps1 (hunk 1 above)
Comment (alexlamtest) [reply tools/TerminalVelocity/Generate-TerminalVelocityFeatures.ps1:49]: Because it already exists on line #177.

**Pull request merged**